mycroft/util/parse.py

```diff
@@ -102,3 +107,56 @@ def normalize_en(text, remove_articles):
         normalized += " " + word
 
     return normalized[1:]  # strip the initial space
+
+
+# TODO: it should be modular in indepent files
```

Comment:
I'm fine with having these in a single file for now.  If the pattern continues for other languages we can implement a single function that splits, removes articles, etc. using just language-specific arrays passed in.